erik/calculator/style/styles.css

```diff
@@ -0,0 +1,112 @@
+body {
+  font-family: "Work Sans", sans-serif;
+}
+.main_div {
+  border: 1px solid black;
+  border-radius: 10px;
+  margin: 0 auto;
+  width: 245px;
+  height: 330px;
+  background: rgb(45, 50, 62);
+}
+.dots {
+  margin: 0px 0px 5px 2px;
+}
+.dot1,
+.dot2,
+.dot3 {
+  margin: 5px 0px 0px 8px;
+  border-radius: 90px;
+  padding: 2px;
+  width: 8px;
+  height: 8px;
+}
+.dot1 {
+  background-color: red;
+}
+.dot2 {
+  background-color: yellow;
+}
+.dot3 {
+  background-color: green;
+}
+.calcInput {
+  margin: 2px;
+}
+
+#output_value {
+  background: rgb(45, 50, 62);
+  border: 1px solid rgb(45, 50, 62);
+  border-radius: 8px;
+  color: white;
+  size: "50";
+  height: 52px;
+  font-size: 50px;
+  width: 235px;
+  margin: 0;
+  text-align: right;
+  font-family: "Sarabun", sans-serif;
+}
+.dots,
+.first_row,
+.second_row,
+.third_row,
+.forth_row,
+.fith_row {
+  display: flex;
+  background-color: rgb(45, 50, 62);
+  border-radius: 8px;
+}
+.clear,
+.pos_neg,
+.precent,
+.divide,
+.btn_numbers,
+.decimal,
+.equal,
+.mulit,
+.minues,
+.add_button {
+  border: 1px solid black;
+  width: 25%;
+  margin: 0px;
+  padding: 10px;
+  text-align: center;
+  font-size: x-large;
+  color: white;
+}
+.divide {
+  width: 83px;
+}
+.btn_zero {
```

Comment:
I'd opt for having a single btn class then all of them can have the same margin/padding etc, and then you can just call out this `btn_zero` for the width. That way there's one single place to go to edit the same styles.